eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -47,6 +50,33 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
+function AddLabels([int] $prNumber)
+{
+  # Adding labels to the pr.
+  if (-not $PRLabel) {
+    Write-Verbose "There are no labels added to the PR."
+    exit 0
+  }
+
+  # Parse the labels from string to array
+  $prLabels = @($PRLabel.Split(",") | % { $_.Trim() } | ? { return $_ })
+  $prLabelUri = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$prNumber"
+  $labelRequestData = @{
+    maintainer_can_modify = $true
+    labels                = $prLabels
```

Comment:
Does anything bad happen if you add a label that already exists?

Reply:
No, I have tested. The script executes successfully without side effect.